server/tilesource/base.py

```diff
@@ -287,58 +548,21 @@ def getRegion(self, width=None, height=None, **kwargs):
         :param height: maximum height in pixels.
         :param **kwargs: optional arguments.  Some options are encoding,
             jpegQuality, jpegSubsampling, top, left, right, bottom,
-            regionWidth, regionHeight, units ('pixels' or 'fraction').
+            regionWidth, regionHeight, units ('pixels' or 'fraction'),
+            magnification, mm_x, mm_y, exact.  See _tileIteratorInfo.
```

Comment:
Units of `pixels` is always in the maximum-resolution layer pixels.  If we want pixels at a specified magnification, then I think we need to have more (and clearer) unit names.  `basepixels` and `magnificationpixels` (seems too long)?

A test I was doing with `getRegion` was to select an ROI via `top`, `left`, `regionWidth`, and `regionHeight`, then scale it to a "nice" size via `width` and `height`.  If, for instance, you had something that annotated the image with an ROI, then this would be useful to make thumbnails of those ROIs.